cli/src/main/scripts/install.sh

```diff
@@ -74,32 +74,45 @@ function install {
 
   PATH_UPDATED=false
   if [[ ! "$LOCATION" == *"$HOME/bin/okta:"* ]]; then
-    [ -f $HOME/.bashrc ] && updateBashPath && PATH_UPDATED=true
-    [ -f $HOME/.zshrc ] && updateZshPath && PATH_UPDATED=true
+    [ -f "$HOME/.bash_profile" ] && updateBashProfilePath && PATH_UPDATED=true
+    [ -f "$HOME/.bashrc" ] && updateBashRcPath && PATH_UPDATED=true
+    [ -f "$HOME/.zshrc" ] && updateZshPath && PATH_UPDATED=true
 
     if [[ ! "$PATH_UPDATED" == "true" ]]; then
-      echoerr "Failed to add \$HOME/bin/okta to your path"
+      echoerr 'Failed to add $HOME/bin/okta to your path'
       exit 1
     fi
   fi
 }
 
 function updateBashPath {
+  # shellcheck disable=SC2016
+  (( $# == 1 )) || { printf 'Usage: updateBashPath $bashPath\n' >&2; return 1; }
+
+  local bashPath=$1
   { # try
-    grep -q 'export PATH=$HOME/bin:$PATH' ~/.bashrc || echo -e '\nexport PATH=$HOME/bin:$PATH' >> ~/.bashrc
+    grep -q 'export PATH=$HOME/bin:$PATH' "$bashPath" || echo -e '\nexport PATH=$HOME/bin:$PATH' >> "$bashPath"
   } || { # catch
     echoerr
-    echoerr 'Failed add $HOME/bin to PATH.  Update your ~/.bashrc will by running the following command:'
-    echoerr '  export PATH=$HOME/bin:$PATH >> ~/.bashrc'
+    echoerr "Failed to add $HOME/bin to PATH.  Update your $bashPath by running the following command:"
+    echoerr "  export PATH=\$HOME/bin:\$PATH >> $bashPath"
   }
 }
 
+function updateBashProfilePath {
+  updateBashPath "$HOME/.bash_profile"
+}
+
+function updateBashRcPath {
+  updateBashPath "$HOME/.bashrc"
+}
+
 function updateZshPath {
   { # try
     grep -q 'export PATH=$HOME/bin:$PATH' ~/.zshrc || echo -e '\nexport PATH=$HOME/bin:$PATH' >> ~/.zshrc
   } || { # catch
     echoerr
-    echoerr 'Failed add $HOME/bin to PATH.  Update your ~/.zshrc will by running the following command:'
+    echoerr 'Failed to add $HOME/bin to PATH.  Update your ~/.zshrc will by running the following command:'
```

Comment:
```suggestion
    echoerr 'Failed to add $HOME/bin to PATH. Update your ~/.zshrc by running the following command:'
```

Reply:
I like the suggestions. As long as I'm touching other code sections, not just the bash stuff, any objection to changin all tildes to $HOME?

Also I might have to do this in a separate PR. I've since changed local `master` to `main` and I honestly don't know if I can do a followup commit with a new branch name.

Reply:
@chrislalos you _should_ be able to do something like `git push <remote> main:master`

Another handy tool is https://cli.github.com/ which can help manage various remotes and create PRs from the command line.